sogs/filtration.py

```diff
@@ -5,9 +5,15 @@
 
 def should_drop_message_with_body(body):
     """return true if we should drop a message given its body"""
+    body = message_body(body)
+    if body is None:
+        return False
```

Comment:
Needs a "Body failed to parse" warning here in case something changes that breaks message parsing in the future.

Reply:
protobuf already does this and it is annoying af